src/Microsoft.Health.Dicom.Api/Features/Formatters/DicomJsonOutputFormatter.cs

```diff
@@ -1,90 +0,0 @@
-﻿// -------------------------------------------------------------------------------------------------
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
-// -------------------------------------------------------------------------------------------------
-
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Text;
-using System.Threading.Tasks;
-using Dicom;
-using Dicom.Serialization;
-using EnsureThat;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Mvc.Formatters;
-using Microsoft.AspNetCore.WebUtilities;
-using Microsoft.Health.Dicom.Core.Features.ChangeFeed;
-using Microsoft.Health.Dicom.Core.Web;
-using Newtonsoft.Json;
-
-namespace Microsoft.Health.Dicom.Api.Features.Formatters
-{
-    public class DicomJsonOutputFormatter : TextOutputFormatter
-    {
-        private readonly JsonDicomConverter _jsonDicomConverter = new JsonDicomConverter(writeTagsAsKeywords: false);
-        private readonly JsonSerializer _jsonSerializer = new JsonSerializer();
-
-        public DicomJsonOutputFormatter()
-        {
-            SupportedMediaTypes.Add(KnownContentTypes.ApplicationDicomJson);
```

Comment:
This formatter triggers for a subset of the JSON media type, and therefore the default json output formatter should trigger and use the configured converter from `fo-dicom`